packages/twenty-server/src/workspace/workspace-query-builder/factories/relation-field-alias.factory.ts

```diff
@@ -111,8 +111,9 @@ export class RelationFieldAliasFactory {
         }
       `;
     }
+
     let relationAlias = fieldMetadata.isCustom
-      ? `${fieldKey}: ${fieldMetadata.targetColumnMap.value}`
+      ? `${fieldKey}: ${referencedObjectMetadata.targetTableName}`
```

Comment:
not sure about this change, I'm facing bugs

Reply:
I'm facing bug even while putting it back actually.
What I'm doing:
- create a test object
- add a record
- go to that record and try to add an activity

Reply:
But I think this change should still be reverted and we should check the targetColumnMap value